### PR TITLE
improved starboard

### PR DESCRIPTION
**Pull request by sourTaste000**

### CODE REVIEW

**Comment by sourTaste000** [comment]
@ilidumb

File: index.js [hunk 1]
```diff
@@ -108,13 +111,14 @@ client.on('messageReactionAdd', async (reaction, user) => {
             return;
         }
     }
-
-    if (reaction.emoji.toString() === "⭐" && reaction.count >= 2) {
+    if(reaction.emoji.toString() === "⭐" && reaction.count >= 3){
         let starEmbed = new Discord.MessageEmbed()
-            .setTitle(`Star message: `)
+            .setTitle(`Star Message from ${reaction.message.author.username}:`)
```
Comment (dominikaaaa) [inline index.js:116]: this is supposed to be the person that sent the original message, no?

Comment (sourTaste000) [reply index.js:116]: yes

Comment (sourTaste000) [reply index.js:116]: this is message sender

**Pull request merged**